server/src/main/java/org/elasticsearch/cluster/routing/allocation/ShardsAvailabilityHealthIndicatorService.java

```diff
@@ -258,7 +259,7 @@ public List<HealthIndicatorImpact> getImpacts() {
                     indicesWithUnavailableReplicasOnly.size() == 1 ? "index" : "indices",
                     getTruncatedIndicesString(indicesWithUnavailableReplicasOnly, clusterMetadata)
                 );
-                impacts.add(new HealthIndicatorImpact(3, impactDescription));
```

Comment:
Was the priority an intentional change? The description of https://github.com/elastic/elasticsearch/issues/84773 has unassigned replicas as a 3.

Reply:
Yes. The description was not updated to reflect that we ended up having only 2 impacts.